radio/src/keys.cpp

```diff
@@ -96,7 +96,8 @@ void Key::input(bool val)
       break;
     case KSTATE_START:
       // TRACE("key %d FIRST", key());
-      putEvent(EVT_KEY_FIRST(key()));
+      if (!IS_SHIFT_KEY(key()))
```

Comment:
Perhaps this test can be removed, I think the only problem we have with the SHIFT key is KEY_REPEAT